lib/config-generator.js

```diff
@@ -314,6 +306,24 @@ class ConfigGenerator {
             .map((plugin) => plugin.plugin);
     }
 
+    buildOptimizationConfig() {
+        const optimization = {
+
```

Comment:
this needs to have `runtimeChunk: 'single'` to extract the runtime (and the list of BC breaks should mention the renaming from `manifest.js` to `runtime.js`)